db/migrate_cards/20180530085450_rename_metric_value_to_metric_answer.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class RenameMetricValueToMetricAnswer < Card::Migration
+  def up
+    update_card :metric_value,
+                name: "Metric Answer", codename: "metric_answer"
```

Comment:
I think it would be my preference to have the *card name* "Answer" rather than "Metric Answer", because there are places where that copy appears on the site.  (Eg on the contributions tab of user pages).

I do think `metric_answer` makes the most sense as a *codename*, though, both because it avoids current conflicts with the answers table and because it may help avoid future conflicts with other mods that have answers.  (For that reason, I like the practice of prefixing codenames with the modname, even though it makes things longer).

Reply:
On the contributions tab it's still "Metric values". I didn't choose `answer` to avoid conflicts with the answer table but it's probably safe as card name.

Reply:
you mean it's still "metric values" even after this change?  (I know it's "metric values" on the live site; I was thinking this would fix that.)